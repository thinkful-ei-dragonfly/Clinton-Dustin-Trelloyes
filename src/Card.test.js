import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';


describe('Card Component', () => {

  it('Renders without Crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI as expected', () =>{
    const tree = renderer
      .create(<Card 
        key={0}
        title={'title'}
        content={'content'}
        /> )
      .toJSON();
      expect(tree).toMatchSnapshot();
  });
});