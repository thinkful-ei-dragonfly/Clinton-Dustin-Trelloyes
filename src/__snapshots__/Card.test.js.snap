// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    title
  </h3>
  <p>
    content
  </p>
</div>
`;
